ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+    # Clear filters and search for the application
+    _clear_filters_and_search_protected_app(acm_obj, app_name, timeout=30)
+
+    # Get DR status locator for this app
+    status_locator = format_locator(acm_loc["protected-app-dr-status"], app_name)
+
+    # Wait for status element to be visible and get actual status
+    acm_obj.wait_for_element_to_be_visible(status_locator, timeout=timeout)
+
+    try:
+        actual_status = acm_obj.get_element_text(status_locator)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)